internal/dag/status_test.go

```diff
@@ -2366,6 +2444,50 @@ func TestDAGStatus(t *testing.T) {
 				},
 			},
 		},
+		"requestHeadersPolicy, Host header invalid on Service": {
+			objs: []interface{}{invalidRequestHeadersPolicyService, serviceKuard},
+			want: map[types.NamespacedName]Status{
+				{Name: invalidRequestHeadersPolicyService.Name, Namespace: invalidRequestHeadersPolicyService.Namespace}: {
+					Object:      invalidRequestHeadersPolicyService,
+					Status:      "invalid",
+					Description: "rewriting \"Host\" header is not supported on a service",
+					Vhost:       invalidRequestHeadersPolicyService.Spec.VirtualHost.Fqdn,
+				},
+			},
+		},
+		"responseHeadersPolicy, Host header invalid on Service": {
+			objs: []interface{}{invalidResponseHeadersPolicyService, serviceKuard},
+			want: map[types.NamespacedName]Status{
+				{Name: invalidResponseHeadersPolicyService.Name, Namespace: invalidResponseHeadersPolicyService.Namespace}: {
+					Object:      invalidResponseHeadersPolicyService,
+					Status:      "invalid",
+					Description: "rewriting \"Host\" header is not supported on response headers",
+					Vhost:       invalidResponseHeadersPolicyService.Spec.VirtualHost.Fqdn,
+				},
+			},
+		},
+		"responseHeadersPolicy invalid on Service": {
+			objs: []interface{}{invalidResponseHeadersPolicyService, serviceKuard},
+			want: map[types.NamespacedName]Status{
+				{Name: invalidResponseHeadersPolicyService.Name, Namespace: invalidResponseHeadersPolicyService.Namespace}: {
+					Object:      invalidResponseHeadersPolicyService,
+					Status:      "invalid",
+					Description: "rewriting \"Host\" header is not supported on response headers",
+					Vhost:       invalidResponseHeadersPolicyService.Spec.VirtualHost.Fqdn,
+				},
+			},
+		},
```

Comment:
these 2 look identical - I'd drop the second since the name for the first is better.

Reply:
I'm such a bad copy/paster at 11pm at night. =(